### ISSUE TITLE
Improve Readme and doc

### ISSUE THREAD

**Issue by Mabsten** [fluttercommunity/rx_command#54]
I really appreciate this package but I find the documentation difficult for newcomers:
- Various discrepancies in the name of the parameters between Readme, methods' signatures and Doc. For example these 3 names are used for the same parameter: `canExecute`, `restriction`, `registration`. I saw that this problem is mentioned in issue [#47 Nullability support](https://github.com/fluttercommunity/rx_command/issues/47#issuecomment-824623306) which cite another different name, `constraint`.
- The Readme seems to be aimed at those who already have some familiarity with this architecture, which is understandable; in the blog https://www.burkharts.net/apps/blog there are articles suitable for newcomers, but they are not up-to-date (for example, reference is made to the Observable of RxDart, which from version 0.23 directly uses the Stream). Not that this is a problem, I just guess that's why these guides weren't promoted more in the Readme.

I am inexperienced and unfamiliar with git but am available to help update the documentation. Personally, the guide that got me aficionado of this package was the blog article https://www.burkharts.net/apps/blog/rxvms-foundations-rxcommand-and-getit/, together with the two articles of the same series preceding it (which is much more complete than "Making Flutter more Reactive" mentioned in the Readme). One idea could be to update this series of articles to the current RxCommand and RxDart Api and make it available in the wiki of this repo.

**Comment by escamoteur** [comment]
Hi,
I completely understand your frustration. It's just a matter of time and energy from my side. Additionally I personally moved to use flutter_command which does no longer rely on streams together with my functional_listener package which offers rx like methods for `ValueListenables`.

It would be awesome if you could help updating the readme here. If you need help with git, no problem, I'm happy to help.

Basically you do a fork (button on the main page of the repo) to your own github account. Then you clone that repository to your machine and make the changes. After that you commit and push the changes back to your copy of this repository. After that you can trigger a PR (pull request) on the github page of your forked repository which allows then me to review and merge your changes.

**Comment by Mabsten** [comment]
Thank you for your answer. I am happy to contribute. Within half a day I will update the readme.

**Comment by Mabsten** [comment]
I have updated the README with https://github.com/fluttercommunity/rx_command/pull/55
I remain available if needed to update the Doc Api (which is much more obsolete than the Readme) and to write some Doc for newcomers (the various parameters emitInitialCommandResult, etc, are not easy to understand even from the doc api)

**Comment by escamoteur** [comment]
Sorry for the delay just merged and pushed with the V6.0.0 many thanks!!!
I would be really happy if you help to improve the docs!
